packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -9795,7 +9840,7 @@ describe('plugin-meetings', () => {
 
```

Comment:
we should add a text to see if nothing breaks if we do not get a numberable string as the exp. For example, 
- if we get 'woops' as the exp. Number('woops') will return NaN. How should the function behave then?
- if we get '{something: 'hello'} as the exp. Number(Object) will return NaN too. How should the function behave then?